docs/index.md

```diff
@@ -0,0 +1,120 @@
+# NEAR Smart Contract SDK for Grain
+
+This project allows you to author smart contracts on the NEAR blockchain using the Grain programming language.
+
+## SDK documentation
+
+View the individual pages for each SDK library:
+
+- [Context](./near/context)
```

Comment:
Do these actually link to something? What is the near/ directory?

Reply:
Yup, the docs for the individual libraries (you can see this in action on the live site). They're in a directory called near because that's what's generated from the structure of the source.